src/Symfony/Component/CssSelector/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ CHANGELOG
 -----
 
  * Add support for `:scope`
+ * Add support for `:is()`
+ * Add support for `:where()`
```

Comment:
This should be moved to new block for 6.4

Reply:
Good catch :wink: